regression/cbmc/r_w_ok_inconsistent_dead/main.c

```diff
@@ -0,0 +1,30 @@
+#include <assert.h>
+#include <stdlib.h>
+
+char *p;
+
+void test_global_pointer()
+{
+  {
+    char c;
+    p = &c;
+  }
+
+  __CPROVER_assume(__CPROVER_r_ok(p, 1));
+  assert(0); // fails
```

Comment:
What does "fails" mean.  Do you really mean unreachable?

Reply:
It means that the assertion fails, i.e. is reachable.